test/dispatch.js

```diff
@@ -0,0 +1,23 @@
+var test = require('tape')
+
+var inu = require('../')
+
+test('Calling dispatch triggers update function with action passed to dispatch.', function (t) {
+  var initialModel = {initial: true}
+  var expectedAction = {type: 'DISPATCHED'}
+  var app = {
+    init: function () {
+      return {model: initialModel}
+    },
+    update: function (model, action) {
+      t.deepEqual(action, expectedAction, 'action passed to update is the action passed to dispatch')
```

Comment:
for these checks you can use `t.equal`, they should be `===` since they should be the same object.